homework-submission/sobhi_updated_week3/src/components/EditUpdateCheck.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+
+class EditUpdateCheck extends React.Component {
+
+    state = {
+        done: this.props.done,
+        updating: false
+    };
+
+    checkboxHandler = () => {
+        this.setState({
+            done: !this.state.done
+        });
+    };
+    toggleUpdating = () => {
+        this.setState({
+            updating: !this.state.updating
+        });
+    };
+
+    render() {
+        const { done, updating } = this.state;
+        const {
+            description,
+            deadline,
+            removeTodo,
+            index,
+            updateTodo
+        } = this.props;
+
+        const updatingForm = (
```

Comment:
Nice to seperate the forms from each others to understand the idea of making each of them, Now it's not required But can you put all of them in a one variable? 😃